spec/core/1.0.md

```diff
@@ -394,12 +403,12 @@ From this and the definition of the `want` message also follows that the use of
 For `want` arguments that are capability names, the corresponding `have` argument is the same string, the capability name.
 A server SHALL NOT agree to using a capability if it has not previously agreed to using the module defining said capability, unless it agrees to using the module in the same `have` message.
 
-For `want` arguments that are module names, the corresponding `have` argument is the module name, with a dot and a positive integer appended to it.
+For `want` arguments that are pairs of module names and major version, the corresponding `have` argument is the corresponding `want` argument, with a dot and a positive integer appended to it.
```

Comment:
"Corresponding" used two times.
Use e. g. "For each `want` argument that is a pair of module names and major version, the corresponding `have` argument is that same `want` argument".